tests/testsuite/artifact_dep.rs

```diff
@@ -1829,7 +1828,18 @@ fn proc_macro_in_artifact_dep() {
 
     p.cargo("check -Z bindeps")
         .masquerade_as_nightly_cargo(&["bindeps"])
-        .with_stderr_data(str![[r#""#]])
+        .with_stderr_data(
+            r#"...
+[UPDATING] `dummy-registry` index
+[LOCKING] 2 packages to latest compatible versions
+[DOWNLOADING] crates ...
+[ERROR] failed to download from `[ROOTURL]/dl/pm/1.0.0/download`
+
+Caused by:
+  [37] Could not read a file:// file (Couldn't open file [ROOT]/dl/pm/1.0.0/download)
```

Comment:
Ok so the deployment is failing because the OS tests expect a slightly different error message. When running locally, this test passes. Do the OS images need to be update?
```diff
- expected
+ actual
Caused by:
- [37] Could not read a file:// file (Couldn't open file [ROOT]/dl/pm/1.0.0/download)
+ [37] Couldn't read a file:// file (Couldn't open file [ROOT]/dl/pm/1.0.0/download)
```

Reply:
You can use the wildcard `[..]` to ignore a section that differs